src/libraries/Microsoft.PowerFx.Interpreter/UDF/UDFLazyBinder.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.PowerFx.Interpreter
 {
     internal class UDFLazyBinder
     {
-        private readonly UserDefinedTexlFunction _function;
+        public readonly UserDefinedTexlFunction _function;
```

Comment:
Why make this field public?

Reply:
So we can read it to make the FunctionInfo

Reply:
Don't make this public. Instead, make a property

public UserDefinedFunction Function => _function;